pyproject.toml

```diff
@@ -92,6 +92,7 @@ torchvision = [
   {markers = "sys_platform != 'darwin' or platform_machine != 'x86_64'", version = "^0"},
   {markers = "sys_platform == 'darwin' and platform_machine == 'x86_64'", version = "~0.17.2"}
 ]
+ocrmac = {markers = "sys_platform == 'darwin'", version = "^1.0.0", optional = true}
 
 [tool.poetry.extras]
 tesserocr = ["tesserocr"]
```

Comment:
Please add an extra names ocrmac as well

Reply:
I added it to extras.